src/lib/telemetry/TelemetryCollector.ts

```diff
@@ -41,9 +42,13 @@ export class TelemetryCollector {
     return errorReport
   }
 
-  public registerMetric() {}
+  public registerMetric(metric: Metric): MetricReport {
+    const metricReport = metricToMetricReport(metric)
+    this.metrics.push(metricReport)
+    return metricReport
+  }
 
-  public async flush(forceRemoteFlush = false) {
+  public async flush(forceRemoteFlush = true) {
```

Comment:
This way every call to `flush` without arguments will remote flush, is this the desired here?